Sources/SWBUtil/Library.swift

```diff
@@ -33,7 +33,7 @@ public enum Library: Sendable {
     @_alwaysEmitIntoClient
     public static func open(_ path: Path) throws -> LibraryHandle {
         #if os(Windows)
-        guard let handle = path.withPlatformString(LoadLibraryW) else {
+        guard let handle = try path.withPlatformString({ p in try p.withCanonicalPathRepresentation({ LoadLibraryW($0) }) }) else {
```

Comment:
I question if `withPlatformString` really helps readability here or not.

Reply:
It's not just for readability, my extension is on UnsafePointer rather than on String like Foundation's (felt it's more flexible that way).